components/script/dom/webglbuffer.rs

```diff
@@ -21,29 +20,72 @@ fn target_is_copy_buffer(target: u32) -> bool {
         target == WebGL2RenderingContextConstants::COPY_WRITE_BUFFER
 }
 
+struct DroppableField {
+    id: WebGLBufferId,
+    marked_for_deletion: Cell<bool>,
+    attached_counter: Cell<u32>,
+    context: WebGLRenderingContext,
```

Comment:
Could `DroppableField` take `WebGLRenderingContext` ?

Reply:
> Could `DroppableField` take `WebGLRenderingContext` ?

I will fix to take `WebGLObject`.

Reply:
No. The fact that this compiles worries me!

Reply:
We can work around this by storing a WebGLMessageSender instead (https://github.com/servo/servo/blob/cf725fc3f59bc1b94e3b35ab4cfe60b3d41559cc/components/script/dom/webglrenderingcontext.rs#L329), since we only use the context for the message sending in https://github.com/servo/servo/blob/cf725fc3f59bc1b94e3b35ab4cfe60b3d41559cc/components/script/dom/webglrenderingcontext.rs#L337-L348.

Reply:
I fixed.
https://github.com/servo/servo/pull/28302/commits/11053ba025b91d1d215c1e3ecd36cdc3f3edec1f
https://github.com/servo/servo/pull/28302/commits/e14c70c6a898b38074af9c7649fe3561f6d0ae31